.github/workflows/python-packaging.yml

```diff
@@ -1,35 +1,30 @@
 name: TileDB-SOMA python sdist & wheels
 
 on:
-  # I.e. on request
+  # trigger via user request, e.g.,
+  #     gh workflow run python-packaging.yml --ref branch
+  # Specify the branch or other ref as required.
```

Comment:
Could you add a note at the top that `workflow_dispatch` pushes to test pypi rather than real pypi?

Reply:
will do